README.adoc

```diff
@@ -112,7 +111,7 @@ https://lists.jboss.org/mailman/listinfo/hawkular-dev[Hawkular developer mailing
 
 We love contributions and pull-requests :-)
 
-To meet the guys, join us on irc://irc.freenode.net/#hawkular[Freenode IRC] in channel #hawkular.
+To chat, join us on irc://irc.freenode.net/#hawkular[Freenode IRC] in channel #hawkular.
```

Comment:
@burmanm, I will add the JIRA link, good suggestions. However, I am not sure I want to add the independently statement that you propose because we no longer advertise Metrics without Alerting.